ejercicio1vulky/src/main/java/com/mycompany/ejercicio1vuky/Pedro.java

```diff
@@ -0,0 +1,17 @@
+
+package com.mycompany.ejercicio1vuky;
+
+
+public class Pedro extends Persona{
+    
+    
+    
+    public void tocarPiano(String cancion){
+       
```

Comment:
Please do some functionality here, you can maybe print a message with the cancion you have received

Reply:
ready